website/docs/advanced/override_grammar/basic.md

```diff
@@ -1,8 +1,10 @@
 ---
-id: command_line_syntax
-title: Command-line syntax
+id: basic
+hide_title: true
+sidebar_label: Basic Override syntax
 ---
-You can modify your configuration via the command line. This includes:
+## Basic Override syntax
+You can manipulate your configuration with overrides (via the command line or the Compose API). This includes:
 - Modifying the the `Defaults List`
 - Modifying the config object
 
```

Comment:
s/matching a Config group/matching a config group

an example here would be nice